docs/features-timeline.md

```diff
@@ -0,0 +1,252 @@
+---
+id: features-timeline
+title: Features Timeline
+---
+
+Which major new features did we introduce in each Babel version? This page includes a very short summary for each _minor_ release, or you can read the full changelog [on GitHub](https://github.com/babel/babel/blob/main/CHANGELOG.md)!
```

Comment:
Good addition 👍 ! I guess can also add a short thing somewhere about new project ideas or repos too since you added polyfills (like if we moved this further back preset-env would be one)

Reply:
```suggestion
Which major new features did we introduce in each Babel version? This page includes a very short summary for each _minor_ release, or you can read the full changelog [on GitHub](https://github.com/babel/babel/blob/main/CHANGELOG.md)!
Additionally, use this timeline to track some other important efforts, such as the [babel-polyfills](https://github.com/babel/babel-polyfills) project.
```